spl/Cargo.toml

```
@@ -35,3 +36,6 @@ spl-associated-token-account = { version = "3", features = ["no-entrypoint"], op
 spl-memo = { version = "4", features = ["no-entrypoint"], optional = true }
 spl-token = { version = "4", features = ["no-entrypoint"], optional = true }
 spl-token-2022 = { version = "3", features = ["no-entrypoint"], optional = true }
+spl-token-group-interface = { version = "0.1.0", optional = true }
```

Let's match the version that's used in the latest `spl-token-2022` crate to reduce the possibility of dependency conflicts: https://github.com/solana-labs/solana-program-library/blob/ad4eb6914c5e4288ad845f29f0003cd3b16243e7/token/program-2022/Cargo.toml#L34

```suggestion
spl-token-group-interface = { version = "0.2.3", optional = true }
```